### ISSUE TITLE
❓ The ratio of training set to evaluating set

### ISSUE THREAD

**Issue by WuS-1997** [RUCAIBox/RecBole#1093]
I use the default config file to set the ratio of data set. In ..\recbole\properties\overall.yaml.
`eval_args: 
  split: {'RS':[0.8,0.1,0.1]}`

but in training epoch the logInfo is
`Train     0: 100%|██████████████████████████| 40/40 [00:00<00:00, 45.22it/s, GPU RAM: 0.03 G/8.00 G]
26 Dec 17:41    INFO  epoch 0 training [time: 0.89s, train loss: 27.6065]
Evaluate   : 100%|███████████████████████| 472/472 [00:01<00:00, 317.63it/s, GPU RAM: 0.03 G/8.00 G]
26 Dec 17:41    INFO  epoch 0 evaluating [time: 1.49s, valid_score: 0.255200]`

I don't know the meaning of these two numbers .Why are the numbers in the  evaluating set(472) larger than in the training set(40).
Did they follow the proportions in the eval_args(0.8,0.1,0.1)? Or other meanings ？
thanks!

**Comment by Wicknight** [comment]
@WuS-1997 Hello, thanks for your attention to RecBole!
First of all, the training data, validation data and test data indeed follow the proportion in the split parameter. 

As for the two numbers 40 and 472, they represent the number of batches in each epoch of the training and verification process respectively. Specifically, the number of batches = the total amount of data / batch_size. However, the total amount of data here is not equal to the split ratio * total data. 

In training phase, the total amount of data = the positive items + the negative items. The positive items = the split ratio * total data, while the negative ones come from the negative sampling module. 

In validation/test phase, the total amount of data depends on your evaluation mode. The total amount of data may be very large if the evaluation mode is full ranking rather than sampled-based ranking.
